Lib/unittest/mock.py

```diff
@@ -1601,7 +1601,11 @@ def _patch_object(
     When used as a class decorator `patch.object` honours `patch.TEST_PREFIX`
     for choosing which methods to wrap.
     """
-    getter = lambda: target
+    if type(target) is str:
+        raise TypeError(f"Target {target} must be a class")
+    else:
```

Comment:
Unnecesary else, the previous if will raise and exit early.

Reply:
removed